includes/class-wc-amazon-payments-advanced-rest-api-controller.php

```diff
@@ -337,29 +336,29 @@ public function authorize( $request ) {
 	 *                                   WP_REST_Response instance.
 	 */
 	public function authorize_and_capture( $request ) {
-		$order_post = $this->is_valid_order( $request['order_id'] );
-		if ( is_wp_error( $order_post ) ) {
-			return $order_post;
+		$or = $this->is_valid_order( $request['order_id'] );
```

Comment:
Can we give this variable a better name, please?

Reply:
done!